README.md

```
@@ -602,6 +603,130 @@ entity.addAttachment({
 });
 ```
 
+### v6.5.0
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)